apps/laboratory/src/utils/ConstantsUtil.ts

```diff
@@ -149,5 +150,7 @@ export const ConstantsUtil = {
   EvmNetworks,
   SolanaNetworks,
   BitcoinNetworks,
-  AllNetworks: [...EvmNetworks, ...SolanaNetworks] as [AppKitNetwork, ...AppKitNetwork[]]
+  AllNetworks: [...EvmNetworks, ...SolanaNetworks] as [AppKitNetwork, ...AppKitNetwork[]],
+  EvmWalletButtons: ['metamask', 'trust', 'coinbase', 'rainbow'] as Wallet[],
+  SolanaWalletButtons: ['metamask', 'trust', 'coinbase', 'jupiter'] as Wallet[]
```

Comment:
At the moment we support top 25 wallets and socials. Eventually we'll support all of the wallets in v2 🙏